quilt/tools/main.py

```diff
@@ -152,7 +154,10 @@ def check_hash(group, hashstr):
         func(**kwargs)
         return 0
     except command.CommandException as ex:
-        print(ex, file=sys.stderr)
+        if str(ex) == command.DEFAULT_QUILT_YML_NOT_FOUND_MSG:
+            parser.print_help()
+        else:
+            print(ex, file=sys.stderr)
```

Comment:
It's a matter for another PR, but we should probably call parser.print_help on all command exceptions, and probably print out the exception message with it.